### PR TITLE
Added bootstrap classes to left bar

### LINKED ISSUE #92: Style leftbar with bootstrap
At present, the leftbar is styled by custom classes, remove these custom classes and add Bootstrap classes there.

### PR DESCRIPTION
**Pull request by Aditya-ds-1806**
fixes #92

### CODE REVIEW

**Comment by Aditya-ds-1806** [comment]
hey, as requested i have styled the input, toggle button. So now the whole left bar is styled with bootstrap classes

**Comment by LalitNM** [comment]
Not accetpable @Aditya-ds-1806. There were a lot more discussion in #93. Add all those changes.

**Comment by Aditya-ds-1806** [comment]
Can you please elaborate? The requirement was to style the left bar with bootstrap classes and I did that. I don't know what else to do

**Comment by Aditya-ds-1806** [comment]
In the discussion, i was asked to style the input and the toggle button, which I did

**Comment by LalitNM** [comment]
I don't think that I have to explain it again, everything is well elaborated there in #93.

Still because you have asked me to elaborate...
`<input>` tag should be like any normal bootstrap's input form, see screenshot=>
image

and the toggle collapse button should be like any bootstrap button, read about bootstrap buttons, and for the division of room name you have to be creative, think about it.

**Comment by Aditya-ds-1806** [comment]
All right, now I understand. Will finish it in half an hour. Thanks!

**Comment by Aditya-ds-1806** [comment]
image
image

This is the result after the recent commit. Is this fine? This is same as bootstrap default. @LalitNM

File: views/index.ejs [hunk 1]
```diff
@@ -26,7 +26,7 @@
             <span class="preview overflow-hidden d-inline-block">Central Lobby</span>
           </li>
         </ul>
-        <button class="btn rounded-0 sidebar-btn" onclick="SidebarToggle()">
+        <button class="btn btn-danger sidebar-btn" onclick="SidebarToggle()">
```
Comment (LalitNM) [inline views/index.ejs]: Do not add `danger` here use `primary`

**Review by LalitNM — CHANGES_REQUESTED** [review]
In addition to the changes requested, there is a bug created by your last commit. Now the search feature of room is not working properly, earlier the search was filtering the name of rooms and showing only the name relevant to search, now it is not filtering the name of rooms.

**Comment by Aditya-ds-1806** [comment]
@LalitNM  Can you check now? I fixed the errors and changed the button to primary

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
Good work @Aditya-ds-1806. Merging this :rocket:

**Comment by Aditya-ds-1806** [comment]
Great, thanks!

**Pull request merged**